integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java

```diff
@@ -278,11 +278,11 @@ public static void replaceStringInFile(String filePath, String regex, String rep
       throws IOException {
     LoggingFacade logger = getLogger();
     Path src = Paths.get(filePath);
-    logger.info("Copying {0}", src.toString());
+    logger.info("Replacing {0}", src.toString());
```

Comment:
```suggestion
    logger.info("Replacing {0}", regex);
```

Reply:
Oops, merged. I will remember this and change it later